pkg/juicefs/juicefs.go

```diff
@@ -177,14 +179,14 @@ func NewJfsProvider(mounter *mount.SafeFormatAndMount) (Interface, error) {
 }
 
 // JfsMount auths and mounts juicefs
-func (j *juicefs) JfsMount(secrets map[string]string, options []string) (Jfs, error) {
+func (j *juicefs) JfsMount(volumeID string, secrets map[string]string, options []string) (Jfs, error) {
 	stdoutStderr, err := j.AuthFs(secrets)
```

Comment:
juicefs client is used by `AuthFs` as well. Better to upgrade before this.

Reply:
I think `version --upgrade` upgrades `juicefs.py` itself too, does it? @davies

Reply:
`juicefs version --upgrade` will upgrade `juicefs` too